clients/java/src/main/java/io/camunda/client/CamundaClient.java

```diff
@@ -1148,6 +1149,22 @@ ActivateAdHocSubProcessActivitiesCommandStep1 newActivateAdHocSubProcessActiviti
    */
   RoleGetRequest newRoleGetRequest(String roleId);
 
+  /**
+   * Command to delete a role by role ID.
+   *
+   * <pre>
+   *
+   * camundaClient
+   *  .newDeleteRoleCommand("123")
```

Comment:
```suggestion
   *  .newDeleteRoleCommand("roleId")
```

🔧 This is a bit more clear than a random number 🙂